hack/gen-certs.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
```

Comment:
We have all of this logic packaged into nice Go libraries already in k8s - might we use that instead of Bash?

Reply:
Sure, we can use that if you think it would be better. I wrote it in bash because `gen-certs.sh` is just a convenience script for local testing. We'd use cert manager in a deployment.